kbs/tools/client/README.md

```diff
@@ -13,7 +13,29 @@ For more sophisticated attestation clients, please refer to [guest components](h
 For help:
 
 ```shell
-./client -h
+kbs-client -h
+```
+
+## Building and installing the client
+
+Build the client binary with support to the default features as:
+
+```shell
+make -C ../../ cli
+```
+
+As an alternative, you can build it with the `sample_only` feature which allow
+to mock attestation, thus, you will be able to perform some operations (e.g.
+get resources) even by running outside of an TEE:
```

Comment:
To be clear, the sample attester is always enabled and you can always use the `kbs-client` outside of a TEE. This flag means that no other attesters are available so you cannot do a real TEE attestation, but you don't have to worry about the dependencies that some of the hw attesters have.

Btw if we build the kbs-client in the CI we probably want to use this feature (or add a new one that disables the resource endpoint completely).

Reply:
If I understand correctly, we do not need `CLI_FEATURES` flag. And we can always build the client-tool with feature `default`. This will support both TEE and non-TEE cases. Right?

Reply:
The default features support all the use cases on all the platforms. For the tests in the CoCo CI, we only use the client for configuration. So we will use the `sample_only` feature there so that we don't have to install the SGX dependencies.

So yes, no need for any additional feature, although maybe at some point we could introduce features to enable/disable the config and resource endpoints separately. Not a high priority.

Reply:
hi @Xynnn007 @fitzthum , sorry, I misunderstood it completely. I built the `default` feature and run kbs-client on a non-TEE environment, and I was getting a attester related error (unfortunately I didn't save the logs). Then I built the client with `sample_only` feature and it magically worked out. Thus, I assumed (wrongly) `sample_only` feature should be used on non-TEE environments.

Now I updated my main branch and I no longer see the error with kbs-client built with `default`...probably I was hitting a bug solved recently.

I will re-phrase the sentences.